### PR TITLE
Add a pdbstore command and fix broken filestore command

### PR DESCRIPTION
**Pull request by DrChat**
Addresses #12.

### CODE REVIEW

File: crates/bin/src/main.rs [hunk 1]
```diff
@@ -90,6 +99,35 @@ fn recursive_listdir(
...
+    // Query the GUID and age.
+    let pdbi = pdb
+        .pdb_information()
+        .context("failed to find PDB information stream")?;
+    let dbi = pdb
+        .debug_information()
+        .context("failed to find DBI stream")?;
+
+    let guid = pdbi.guid;
+    let age = dbi.age().unwrap_or(pdbi.age); // FIXME: Is this right?
```
Comment (comcma) [inline crates/bin/src/main.rs:124]: Can you make an issue for this? What is the confusion here?

File: crates/bin/src/main.rs [hunk 2]
```diff
@@ -100,7 +138,7 @@ fn get_file_path(filename: &Path) -> anyhow::Result<String> {
         .context("Failed to convert file name")?;
 
     let filestr = format!(
-        "filestore/{}/{:08x}{:x}/{}",
+        "{}/{:08x}{:x}/{}",
```
Comment (comcma) [inline crates/bin/src/main.rs:141]: Are people going to be expecting the previous behavior? Should we rev the version?

File: crates/bin/src/main.rs (hunk 1 above)
Comment (comcma) [reply crates/bin/src/main.rs:124]: Is there a way to test this with pdblisters own pdb generated after build?

Comment (DrChat) [reply crates/bin/src/main.rs:124]: Not sure which field is written to the folder name, or if symchk follows the same logic here. Will check and see.

File: crates/bin/src/main.rs (hunk 2 above)
Comment (DrChat) [reply crates/bin/src/main.rs:141]: Heh the previous version was broken, so it seems that nobody actually uses this as of now :)

File: crates/bin/src/main.rs [hunk 3]
```diff
@@ -585,14 +637,42 @@ async fn run() -> anyhow::Result<()> {
         }
         Some(("filestore", matches)) => {
             /* List all files in the directory specified by args[2] */
-            let dir = Path::new(matches.get_one::<String>("fielpath").unwrap());
```
Comment (DrChat) [inline crates/bin/src/main.rs:588]: Fyi: Typo that was fixed in this PR

File: crates/bin/src/main.rs (hunk 1 above)
Comment (DrChat) [reply crates/bin/src/main.rs:124]: Confirmed that symchk follows this logic, so we should be good :)

**Review by comcma — APPROVED** [review]

**Pull request merged**